packages/docs/src/components/sidebar/navigationRoutes.ts

```diff
@@ -342,6 +342,10 @@ export const navigationRoutes: NavigationRoute[] = [
         name: 'dropdown',
```

Comment:
It's advantageous to use minimal amount of items to test your component.

I.e. if for all cases you only need 2 items (1 category and 1 item) - then we should have 2 items. But solution is yours and if you deem important to keep that number - so be it.

<img width="564" alt="image" src="https://user-images.githubusercontent.com/5394573/183218049-6783e609-f8b4-4567-b68c-afff51dfddc5.png">

Reply:
I've used examples from the Figma design, where is given a lot of use cases. And I think better to have more examples with different variations to see if something was broken during the development.

Reply:
All right, up to you. :)